alpha/lib/model/entry.php

```diff
@@ -351,41 +351,49 @@ public static function getMultiLingualSupportedFields()
 		return self::$multiLingualSupportedFields;
 	}
 	
-	public function setName($v)
+	/**
+	 * @param $v
+	 * @param $fieldName
+	 * @return mixed|string|null
+	 * @throws KalturaAPIException
+	 *
+	 * This function calculates the value that is to be set in the relevant field in the db (called defaultValue
+	 * It also updates the multilingual object of the entry if applicable
+	 */
+	protected function getValueToSetInDbAndUpdateMultiLangObject($v, $fieldName)
 	{
-		$name = $v['default'];
-		if (multiLingualUtils::isMultiLingualRequest($v))
+		if(is_null($v))
+		{
+			return $v;
+		}
+		$multiLingualValue = (is_string($v)) ? multiLingualUtils::getMultiLingualStringArrayFromString($v)->toObjectsArray() : $v;
```

Comment:
better change to
$multiLingualValue = $v;
if(is_string($v))
...

also might worth dividing the call for the twop functions